### DISCUSSION TITLE
Where is the source code for the Android app?

### DISCUSSION THREAD

**Discussion by Darin755** [netbirdio/netbird#1419] [Q&A / Support] [answered]
I have seen multiple posts about the Foss nature of the android app but I'm still waiting for the source code. 

Does anyone know when I can get access to the source code? Can I get a update on what's the current status?

**Comment by mlsmaycon** [comment]
Hello, @Darin755, can you share your use case for it?

**Reply by Darin755** [reply]
Mostly I am looking to build a privacy and Foss friendly version free of non-free dependencies. I'm actually pretty new at this and I figured I'd get a repo going and maybe at some point get it on F-droid.

It would involve a bit of a learning curve but I think it would be worth it in the end.

**Reply by mlsmaycon** [reply, marked as answer]
Thanks for sharing your use case @Darin755. We are working with the end of January as the date to open-source the code. We have to clean and document the code before. 

Regarding the non-free dependencies, just to give you a heads up, we are using Firebase for crash and analytics, with no user data. We are fine to have support to disable it as an option in the app.

**Reply by Darin755** [reply]
Thanks, Its nice to know that the release is just down the road.
